packages/cfd/src/Containers/cfd-server-maintenance-modal.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Button, Modal } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
+import { isDesktop, CFD_PLATFORMS } from '@deriv/shared';
+
+const CFDServerMaintenanceModal = observer(() => {
+    const { modules, common } = useStore();
+    const { cfd } = modules;
+    const { platform } = common;
+
+    const { setServerMaintenanceModal, is_server_maintenance_modal_visible } = cfd;
+
+    const getMaintenanceTime = () => {
+        switch (platform) {
+            case CFD_PLATFORMS.DXTRADE:
+                return '08:00 GMT';
+            case CFD_PLATFORMS.CTRADER:
+                return '10:00 GMT';
+            case CFD_PLATFORMS.MT5:
+            default:
+                return '03:00 GMT';
+        }
+    };
```

Comment:
In my opinion this is much more performant and also you can place it outside the component.
```
const maintenance_time = {
    [[CFD_PLATFORMS.DXTRADE]]: '08:00 GMT',
    [[CFD_PLATFORMS.CTRADER]]: '10:00 GMT',
    [[CFD_PLATFORMS.MT5]]: '03:00 GMT',
};
```